{"timestamp":"2025-08-13T15:56:15.234359","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-13T15:56:15.235571","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/etl_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-13T15:56:16.263927Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.265802Z","level":"info","event":" Previous state of the Task instance: queued","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.267134Z","level":"info","event":"Current task name:load_genome_tags","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.268143Z","level":"info","event":"Dag name:csv_ingestion_split_dag","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:15.900742","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-08-13T15:56:16.365613","level":"warning","event":"Skipping masking for a secret as it's too short (<5 chars)","logger":"airflow.sdk.execution_time.secrets_masker"}
{"timestamp":"2025-08-13T15:56:16.377940","level":"info","event":"Connection Retrieved 'my_postgres'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-13T15:56:16.453310","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OperationalError","exc_value":"connection to server at \"postgres\" (172.18.0.4), port 5432 failed: FATAL:  password authentication failed for user \"airflow\"\n","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":825,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1088,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/modules/load_data.py","lineno":52,"name":"load_csv_to_postgres"},{"filename":"/opt/airflow/modules/load_data.py","lineno":19,"name":"get_postgres_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py","lineno":122,"name":"connect"}]}]}
{"timestamp":"2025-08-13T15:56:16.467859Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.470432Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.475335Z","level":"info","event":"Task:<Task(PythonOperator): load_genome_tags>","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.488095Z","level":"info","event":"Failure caused by connection to server at \"postgres\" (172.18.0.4), port 5432 failed: FATAL:  password authentication failed for user \"airflow\"","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-13T15:56:16.491009Z","level":"info","event":"","chan":"stdout","logger":"task"}
